sirepo/srunit.py

```diff
@@ -99,6 +99,13 @@ def sim_data(sim_name=None, sim_type=None, sim_types=CONFTEST_DEFAULT_CODES, cfg
     return fc.sr_sim_data(sim_name=sim_name, sim_type=sim_type), fc
 
 
+@contextlib.contextmanager
+def srcontext():
```

Comment:
I think it is problematic naming this the same name as a module. That aside, I think this is now `auth_db_session()`.

Reply:
fc84edd7